src/utils/ContractsEventsSubscription.js

```diff
@@ -12,6 +12,10 @@ async function getPastEvents(contract, fromBlock, toBlock) {
   const batchCount = Math.ceil((toBlock - fromBlock) / BATCH_SIZE);
```

Comment:
you are doing math with `fromBlock` and `toBlock` here. How come they aren't integers at this point?

Reply:
I don't know. I havent looked deeper because I saw that after restarting first mainnet sentinel and I wanted a quick fix 😁

Reply:
string comes from here, actually: https://github.com/leapdao/leap-node/blob/master/src/bridgeState.js#L75